test/system/roles/public_health/dashboard/import_verifier.rb

```diff
@@ -41,295 +42,431 @@ class PublicHealthMonitoringImportVerifier < ApplicationSystemTestCase
   }.freeze
   ADDRESS_FIELDS = %i[address_line_1 address_city address_zip address_line_2].freeze
 
-  def verify_epi_x_field_validation(jurisdiction, workflow, file_name)
```

Comment:
I actually ended up removing this constants because it's not being used anymore, great catch! https://github.com/SaraAlert/SaraAlert/pull/1170/commits/feb3078e51a1da7c5235d18fec6a2657f7ea38d3